src/differentials.jl

```diff
@@ -147,7 +145,7 @@ is not defined.
 """
 struct DNE <: AbstractDifferential end
 
-extern(x::DNE) = error("`DNE` cannot be converted into an external type.")
+extern(x::DNE) = throw(ArgumentError("`DNE` cannot be converted to an external type."))
```

Comment:
Explain that this is because it does not exist

Reply:
done!